.github/workflows/pr_env.yaml

```diff
@@ -22,6 +22,13 @@ jobs:
         run: echo "create_env=${{ contains(fromJson(vars.PR_ENV_ALLOW_LIST), github.event.sender.login) && contains(github.event.pull_request.labels.*.name, vars.PR_ENV_LABEL) }}" >> "$GITHUB_OUTPUT"
 
   build-and-push-pr-image:
+    strategy:
```

Comment:
Why this change? I don't think we need we need an additional temporary image per build.
I guess this is a side-effect of not having the operations enabled in prod.
Let's wait until having them once we have the release

Reply:
Yes, I don't intend to merge changes to the pipeline. Was trying things out.
Will revert it